onnxruntime/core/optimizer/qdq_transformer/selectors_actions/shared/utils.cc

```diff
@@ -0,0 +1,155 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include <iostream>
+#include <string>
+#include <vector>
+#include <optional>
+
+#include <core/graph/graph_viewer.h>
+#include <core/providers/common.h>
+
+#include "core/optimizer/qdq_transformer/selectors_actions/qdq_selector_action_transformer.h"
+#include "core/optimizer/qdq_transformer/selectors_actions/qdq_selectors.h"
+
+#include "utils.h"
```

Comment:
this one should be first

Reply:
sorry, to clarify, I meant it should be the very first include. the rationale for this is to ensure that the related header can be included on its own and not require other external includes before it.